frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -479,3 +479,9 @@ button.usa-pagination__button.usa-button {
     }
   }
 }
+
+.icon-btn {
+  display: flex;
```

Comment:
a couple of thoughts here if you have time for css:
* I think that you could accomplish all of this by wrapping the link in an unstyled button. If you do a find for "unstyled" in the project you should be able to find other examples of where we've done this. 
* In case that doesn't work, I think you can at least move the `display: flex` to a `display-flex` class and remove it from this custom css. `align-items`, probably not, I'm not sure about `gap`, but that could maybe be handled with padding or margin instead?

Reply:
@doug-s-nava because the rendering of the icon on this button is inconsistent and this button utilizes nextJS's Link, I don't think an unstyled button could be used in this case. Unless I'm missing something? Here's the link to the code in reference. 

https://github.com/HHS/simpler-grants-gov/blob/3f926bcfd691e0e8f1906b31f7dd2fca63325484/frontend/src/components/Header.tsx#L60